### PR TITLE
Training updates

### PR DESCRIPTION
**Pull request by salsa-nathan**
This PR contains updates for the following Jira stories:

- DQL-36 - Add any data.qld customisations
- DQL-42 - Tidy-up - CKAN 2.8.2 upgrade remediations
- DQL-13 - Create new CKAN theme extension
- DQL-19 - Modify CKAN dataset schema
- DQL-47 - Google Analytics - Identify what dataset/resources belong to which organisations

Please note: Some updates for these stories will already exist in the `develop` branch as at the point it was branched to `training`.

### CODE REVIEW

File: ckanext/data_qld_theme/templates/datarequests/base.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{% block secondary_content %}
+
+  <section class="module module-narrow module-shallow">
+    <h2 class="module-heading"><i class="icon-info-sign fa fa-info-circle"></i>{% trans %}Data request{% endtrans %}</h2>
+    <div class="module-content">
+      <p>
+        {% trans %}Data requests allow registered users to ask for data that is not currently available.{% endtrans %}
+      </p>
+      <p>
+        {% trans %}Prior to making a data request, please use the search and filter tools to attempt to locate the desired data from among the published datasets. You can also review the exisiting data requests as other users may have made requests for similar data.{% endtrans %}
```
Comment (ThrawnCA) [inline ckanext/data_qld_theme/templates/datarequests/base.html]: Misspelled "existing"

File: ckanext/data_qld_theme/templates/user/new.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+  {% block help_inner %}
+  <h2 class="module-heading">{{ _('Why Sign Up?') }}</h2>
+  <div class="module-content">
+    <p>{% trans %}Users must register to make data requests, comment against data requests and to give dataset feedback.{% endtrans %}</p>
```
Comment (ThrawnCA) [inline ckanext/data_qld_theme/templates/user/new.html]: Inconsistent use of "to"; the second list term omits it but the third includes it.

File: ckanext/data_qld_theme/templates/datarequests/base.html (hunk 1 above)
Comment (MarkCalvert) [reply ckanext/data_qld_theme/templates/datarequests/base.html]: Fixed spelling mistake

File: ckanext/data_qld_theme/templates/user/new.html (hunk 2 above)
Comment (MarkCalvert) [reply ckanext/data_qld_theme/templates/user/new.html]: This was copied from AC 1. b. i. in the story https://salsadigital.atlassian.net/browse/DQL-50

**Pull request merged**